README.md

```diff
@@ -20,6 +20,9 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `RAW_EVENT_MESSAGE`: Setting `line` to Raw `event.message` *(Optional)*:
+        * It can be enabled by setting `RAW_EVENT_MESSAGE` to `YES` or `TRUE`
+        * Enabling it moves the following `event`-related `meta` data from the `line` field to the `meta` field
```

Comment:
Might also want to include that it defaults to false

Reply:
Nice suggestion! Added!